Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-02-06
Forwarded: no
Description: prevent breaking of source tree
 Without this patch abiword breaks its own source tree
 so the second build fail


--- a/Makefile.am
+++ b/Makefile.am
@@ -54,10 +54,6 @@
 
 DISTCLEANFILES = \
 	abiword-$(ABIWORD_SERIES).pc \
-	plugin-builtin.m4 \
-	plugin-configure.m4 \
-	plugin-list.m4 \
-	plugin-makefiles.m4 \
 	doltcompile \
 	doltlibtool
 
--- a/src/wp/test/Makefile.am
+++ b/src/wp/test/Makefile.am
@@ -19,4 +19,4 @@
 #include $(top_srcdir)/includes.mk
 
 SUBDIRS= xp @PLATFORM@
-DIST_SUBDIRS = cocoa unix win xp
+DIST_SUBDIRS = unix xp
